src/modules/auth/components/shared/googleAuthButton/googleAuthButton.tsx

```diff
@@ -3,19 +3,16 @@ import googleIcon from '#/modules/auth/assets/icons/google.svg';
 import styles from './googleAuthButton.module.css';
 
 export function GoogleAuthButton() {
-  function handleClick(e: React.MouseEvent<HTMLButtonElement>) {
-    console.log(e.target);
-    console.log('continue with google');
-  }
-
   return (
-    <button className={styles.wrapper} onClick={handleClick}>
-      <Box className={styles.container}>
-        <Box className={styles.logo}>
-          <img src={googleIcon} alt="Google Icon" />
-        </Box>
-        <Box className={styles.text}>Continue with Google</Box>
+    <form action={`${import.meta.env.VITE_API_URL}/auth/google/login`}>
+      <Box className={styles.wrapper}>
+        <button type="submit" className={styles.container}>
+          <Box className={styles.logo}>
```

Comment:
can you use sx for styling. 
you can use this as an example: https://github.com/vention-js-lab/ventioners-stayhub-frontend/pull/5#discussion_r1838030972 .

Reply:
I will soon create a separate PR for that